dotcom-rendering/src/client/index.scheduled.ts

```diff
@@ -9,6 +9,7 @@ import './webpackPublicPath';
 
 import type { ScheduleOptions } from '../lib/scheduler';
 import { schedule, setSchedulerConcurrency } from '../lib/scheduler';
+import { identity } from '../lib/useAuthStatus';
```

Comment:
I'm not sure the resulting file name changes are worth it, but it feels a bit funny to be importing a hook module outside of a React component. I don't think it really matters because we're not trying to use the hook!